rust/osm2lanes/src/transform/lanes_to_tags.rs

```diff
@@ -84,6 +84,54 @@ pub fn lanes_to_tags(lanes: &[Lane], locale: &Locale, config: &Config) -> TagsRe
     }) {
         tags.checked_insert("oneway", "yes")?;
         oneway = true;
+    } else {
+        // Forward
+        let forward_lanes = lanes
+            .iter()
+            .filter(|lane| {
+                matches!(
+                    lane,
+                    Lane::Travel {
+                        designated: Designated::Motor | Designated::Bus,
+                        direction: Some(Direction::Forward),
+                        ..
+                    }
+                )
+            })
+            .count();
+        tags.checked_insert("lanes:forward", forward_lanes.to_string())?;
+        // Backward
+        let backward_lanes = lanes
+            .iter()
+            .filter(|lane| {
+                matches!(
+                    lane,
+                    Lane::Travel {
+                        designated: Designated::Motor | Designated::Bus,
+                        direction: Some(Direction::Backward),
+                        ..
+                    }
+                )
+            })
+            .count();
+        tags.checked_insert("lanes:backward", backward_lanes.to_string())?;
+        // Both ways
+        if lanes.iter().any(|lane| {
+            matches!(
+                lane,
+                Lane::Travel {
+                    designated: Designated::Motor,
+                    direction: Some(Direction::Both),
+                    ..
+                }
+            )
+        }) {
+            tags.checked_insert("lanes:both_ways", "1")?;
+            // TODO: add LHT support
```

Comment:
emit an unimplemented warning if you want to leave this as is?

Reply:
Got it.